engine/lib/river.php

```diff
@@ -808,9 +815,9 @@ function _elgg_river_enable($event, $type, $entity) {
 	LEFT JOIN {$dbprefix}entities AS te ON te.guid = rv.target_guid
 	SET rv.enabled = 'yes'
 	WHERE (
-			(se.enabled = 'yes' OR se.guid IS NULL) AND 
+			(se.enabled = 'yes' OR se.guid IS NULL) AND
```

Comment:
the space at the end of the line? that was removed by atom, the editor i am using. as far as i am aware, such spaces play no role in the parsing of SQL. i have tested these changes in the plugin i am coding that requires these changes to work and have not found any bugs or errors yet. the extent of my testing is not exhaustive, but i have also never seen any requirement for such spaces in SQL.